pkgs/applications/blockchains/bitcoin.nix

```diff
@@ -14,6 +18,16 @@ stdenv.mkDerivation rec{
     sha256 = "5c7d93f15579e37aa2d1dc79e8f5ac675f59045fceddf604ae0f1550eb03bf96";
   };
 
+  desktop = optional withGui (fetchurl {
+    url = "https://raw.githubusercontent.com/bitcoin-core/packaging/${majorVersion}/debian/bitcoin-qt.desktop";
+    sha256 = "0cpna0nxcd1dw3nnzli36nf9zj28d2g9jf5y0zl9j18lvanvniha";
+  });
+
+  pixmap = optional withGui (fetchurl {
+    url = "https://raw.githubusercontent.com/bitcoin/bitcoin/v${version}/src/qt/res/src/bitcoin.svg";
```

Comment:
It would be better to use

```
url = "https://raw.githubusercontent.com/bitcoin/bitcoin/v${version}/share/pixmaps/bitcoin128.png";
```

Reply:
> So we don't install an SVG file with the PNG extension.

oops I missed this in review. ideally we would just fix the `install` command to use the proper filename.

Reply:
Ok, i fixed url